bin/telemetry_sim_ComCam

```diff
@@ -142,6 +162,21 @@ if __name__ == "__main__":
         else:
             imageName = "{}_{}_{}_{:06d}".format(args.telcode, args.controller, DAYOBS, seqN)
 
+        # The START telemetry for Airmass using ATPtg.currentTargetStatus
```

Comment:
Remove AT from the various comments in this section as this is a MT simulation.

Reply:
Fixed